src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -2,10 +1,42 @@
+import { ChangeEvent } from 'react';
+
+import { FilterOption } from '../../enums/filter-options';
+
+type Props = {
+  filterOption: FilterOption;
+  searchQuery: string;
+  onFilterTodos: (event: ChangeEvent<HTMLSelectElement>) => void;
+  onSearch: (event: ChangeEvent<HTMLInputElement>) => void;
+  onResetSearch: () => void;
+};
+
+const filterOptionsArray = Object.values(FilterOption);
+
+export const TodoFilter: React.FC<Props> = ({
+  filterOption,
+  searchQuery,
+  onFilterTodos,
+  onSearch,
+  onResetSearch,
```

Comment:
The naming of the event handlers could be improved. It's recommended to use the `handle` prefix for methods that are used as event handlers. For example, `onFilterTodos` could be renamed to `handleFilterTodos`, `onSearch` to `handleSearch`, and `onResetSearch` to `handleResetSearch`. This makes it clear that these methods are event handlers. ([here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) are some naming conventions for methods)